testsuite/reaction_ensemble.py

```diff
@@ -93,28 +96,43 @@ def test_ideal_titration_curve(self):
 
         volume = ReactionEnsembleTest.volume
         average_NH = 0.0
+        average_NHA = 0.0
+        average_NA = 0.0
         average_degree_of_association = 0.0
         num_samples = 1000
         for i in range(num_samples):
             RE.reaction()
-            average_NH += system.number_of_particles(
-                type=type_H)
+            average_NH += system.number_of_particles( type=type_H)
+            average_NHA += system.number_of_particles( type=type_HA)
+            average_NA += system.number_of_particles( type=type_A)
             average_degree_of_association += system.number_of_particles(
                 type=type_HA) / float(N0)
         average_NH /= num_samples
+        average_NA /= num_samples
+        average_NHA /= num_samples
         average_degree_of_association /= num_samples
         pH = -np.log10(average_NH / volume)
-        K_apparent_HA_diss = K_HA_diss * standard_pressure_in_simulation_units / temperature
-        pK_a = -np.log10(K_apparent_HA_diss)
-        print(average_degree_of_association)
-        real_error_in_degree_of_association = abs(
-            average_degree_of_association - ReactionEnsembleTest.ideal_degree_of_association(
-                pK_a, pH)) / ReactionEnsembleTest.ideal_degree_of_association(
-            pK_a, pH)
+        pK_a = -np.log10(K_HA_diss)
+        #ideal=ReactionEnsembleTest.ideal_degree_of_association( pK_a, pH);
+        ideal=0.05080885682460401; # hard-coded value from the original setup by Jonas
```

Comment:
I would not hard code these values but use the function that determines these values